optuna/study.py

```diff
@@ -149,6 +149,15 @@ def trials(self) -> List[FrozenTrial]:
 
         return self.get_trials(deepcopy=True, states=None)
 
+    def _is_multi_objective(self) -> bool:
```

Comment:
I'm not confident but I'm wondering whether this method should be public, as it could be invoked from outside the class (e.g. sampler, pruner). I want to hear other opinions.

Reply:
IMO, this method is only invoked by developers, so it should be private to align other codes of Optuna (e.g. `Study._ask`, `Study._tell`, or `Study._log_completed_trials`)

Reply:
Thank you for the comments both. I will keep it private `_is...`.